readthedocs/search/management/commands/reindex_elasticsearch.py

```diff
@@ -52,6 +51,9 @@ def _run_reindex_tasks(self, models, queue):
 
             index_name = doc._doc_type.index
             new_index_name = "{}_{}".format(index_name, timestamp)
+            # Set index temporarily for indexing,
+            # this will only get set during the running of this command
+            doc._doc_type.index = new_index_name
```

Comment:
This is only run in a management command, not inside a long running process.

Reply:
@ericholscher I understand. But this management command will override the index name only to the instance where you are running the command. But it will not change the index name where the indexing task will run. So I suspect the celery task will not get the new index name while running the indexing tasks and maybe raise error or index into the old index.